.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build and lint
+
+on: [push]
+
+jobs:
+  build-node:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: set up node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Comment:
Remove.

Reply:
![Screenshot 2023-11-19 at 22 39 52](https://github.com/graasp/graasp-mobile-builder/assets/67112345/7fa16c1d-efda-46c7-8358-c51db63bc462)
How can we solve this?

Reply:
Why is it using yarn v1 ?
Update to latest version of yarn: `yarn set version latest` and then commit the files generated.